README.md

```diff
@@ -147,55 +122,55 @@ Each SDK will auto generate API reference documentation using the respective lan
+New test vectors should follow the standard [vector structure](./test-vectors/). Vectors are automatically validated against their [JSON Schema](https://json-schema.org/) via CI.
 
-Create a PR in this repo for adding / updating web5 test vectors
+Create a PR in this repo for adding / updating Web5 test vectors.
 
 ### Feature Completeness By SDK
 
-Test vectors are also used to determine feature completeness via our [test harness](./test-harness/README.md). Results of test harness runs can be found [here](https://tbd54566975.github.io/sdk-development/).
-
 ## Web5 SDK Features
 
+Test vectors are used to determine feature completeness via our [test harness](./test-harness/README.md). Results of test harness runs can be found [here](https://tbd54566975.github.io/sdk-development/).
+
 ### Cryptographic Digital Signature Algorithms (DSA)
 
-| Algorithm       | Typescript | Kotlin | Rust | Swift |
-| --------------- | ---------- | ------ | ---- | ----- |
-| `ES256K`        | ✅          | ✅      | ✅    | ❌     |
-| `EdDSA:Ed25519` | ✅          | ✅      | ✅    | ❌     |
+| Key Type                                          | Algorithm                                                             | Type                     |
+| ------------------------------------------------- | --------------------------------------------------------------------- | ------------------------ |
+| [secp256k1](https://en.bitcoin.it/wiki/Secp256k1) | [`ES256K`](https://datatracker.ietf.org/doc/html/rfc8812#section-3.1) | Signing and Verification |
+| [Ed25519](https://ed25519.cr.yp.to/)              | [`EdDSA`](https://datatracker.ietf.org/doc/html/rfc8032)              | Signing and Verification |
 
 > [!IMPORTANT]
-> In-memory signing using Secp256k1 **MUST** produce k-deterministic low-s signatures. Verification **must not require** low-s signatures
+> In-memory signing using secp256k1 **MUST** produce k-deterministic low-s signatures with [ECDSA](https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm) as per [RFC6979](https://www.rfc-editor.org/rfc/rfc6979). Verification **must not require** low-s signatures.
 
 ### Key Management
 
 Each SDK will implement a consistent and extensible _public interface_ for key management minimally providing concrete implementations of:
 
-| Feature               | Typescript | Kotlin | Rust | Swift |
-| --------------------- | ---------- | ------ | ---- | ----- |
-| Key Manager Interface | ❌          | ✅      | ✅    | ❌     |
-| In-Memory Key Manager | ❌          | ✅      | ✅    | ❌     |
-| AWS KMS               | ❌          | ✅      | N/A  | N/A   |
-| Device Enclave        | N/A        | ❌      | N/A  | ❌     |
-| Keychain              | N/A        | ❌      | N/A  | ❌     |
+| Feature               |
+| --------------------- |
+| Key Manager Interface | 
+| In-Memory Key Manager |
+| AWS KMS               |
+| Device Enclave        |
+| Keychain              |
```

Comment:
Good question @jiyoontbd 

I'm wondering if we should even include `Device Enclave`?  I'm a newb when it comes to native mobile dev, but to my knowledge we can't store `Ed25519` or `secp256k1` keys in any iOS or Android device's enclave (only NIST `P-256`/`secp256r1` is supported).  Perhaps we leave out requiring the implementation of a Key Manager for mobile device secure enclaves?

On `Keychain`: Perhaps include both iOS and Android?  As noted [here](https://developer.apple.com/documentation/security/certificate_key_and_trust_services/keys/protecting_keys_with_the_secure_enclave) and implemented [here](https://github.com/TBD54566975/web5-swift/blob/main/Sources/Web5/Crypto/KeyManager/Local/KeychainKeyManager.swift), it is possible to stored Ed25519 and secp256k1 keys in iOS Keychain.

Do we already have a preferred approach for Android? AFAIK Android Key Store has no support for Ed25519 or secp256k1.  Perhaps an approach similar to the one [Samsung took](https://developer.samsung.com/blockchain/keystore/understanding-keystore/what-makes-keystore-unique.html)?

Reply:
so right now we only have inmemory + aws kms (in all languages i think?), and ios keychain (swift only). 

currently i'm not sure if android key management _or_ general mobile device enclave should be listed in this column since:
1) we haven't implemented them yet and 
2) we also have not decided how to write a key manager against either android phones or device enclaves

created issues here 
https://github.com/TBD54566975/tbdex/issues/271 
https://github.com/TBD54566975/tbdex/issues/272 

please feel free to close these if needed, i am probably wrong / not up to date on this subject.